setup.cfg

```diff
@@ -49,6 +49,7 @@ python_requires = >=3.8
 # Updates here should be reflected in `docs/requirements.txt`
 install_requires =
     apache-airflow~=2.4
+    sqlalchemy<2.0
```

Comment:
Is this a necessary requirement?

Reply:
I added a comment to clarify why I added this. It's in response to a warning from SQLAlchemy ([source](https://sqlalche.me/e/b8d9)).